prediction_market_agent/agents/arbitrage_agent/data_models.py

```diff
@@ -0,0 +1,73 @@
+import typing as t
+
+from prediction_market_agent_tooling.markets.agent_market import AgentMarket
+from pydantic import BaseModel, computed_field
+
+
+class Correlation(BaseModel):
+    near_perfect_correlation: bool
+    reasoning: str
+
+
+class CorrelatedMarketPair(BaseModel):
+    main_market: AgentMarket
+    related_market: AgentMarket
+
+    def __repr__(self) -> str:
```

Comment:
```suggestion
    def __str__(self) -> str:
```

Reason:

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/77b1c9cc-7945-4e89-a4a3-fa85e2290e16)